haxepunk/graphics/Tilemap.hx

```diff
@@ -551,6 +495,34 @@ class Tilemap extends Canvas
 	}
 
 	/**
+	 * The tinted color of the Canvas. Use 0xFFFFFF to draw the it normally.
+	 */
+	public var color(get, set):Color;
+	function get_color():Color return _color;
+	function set_color(value:Color):Color
```

Comment:
Seems reasonable. There might be an argument for doing the same with width/height.